Taskfile.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+dotenv: ['.env']
+
+tasks:
+  run:
+    cmds:
+      - go run main.go
+  dev_server:
```

Comment:
The task 'dev_server' is executing a specific version of the 'air' package. While it's good practice to pin versions, ensure that this version is still actively maintained and consider documenting the reason for choosing this specific version to avoid potential issues in the future if there are breaking changes or vulnerabilities in later versions.